swift-ticketing/src/swift_ticketing/handlers.clj

```diff
@@ -105,21 +106,23 @@
   (validate-req booking-id ::specs/booking-id #(cancel-booking db-spec booking-id)))
 
 (defn- get-booking-status [db-spec uid booking-id]
-  (let [result (:booking/booking_status (jdbc/execute-one! db-spec (booking/get-booking-status uid booking-id)))]
+  (let [result (-> (booking/get-booking-status db-spec uid booking-id)
+                   first
```

Comment:
If the db function is only going to fetch one booking's status, it can extract that from the record collection itself so that all callers don't have to do this.